### PR TITLE
Break core.cljs into more cohesive files

### PR DESCRIPTION
**Pull request by ivansalazar**

### CODE REVIEW

File: src/qwircl/grid.cljs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+(ns qwircl.grid)
+
+(defn with-positions [grid positions hand]
+  (reduce #(assoc-in %1 (:coordinates %2) (hand (first (:hand %2)))) grid positions))
+
+(def transformations {:horizontal {:x [dec inc] :y [identity identity]}
+                      :vertical {:x [identity identity] :y [dec inc]}})
+(defn get-neighbors 
+  ([grid [x y] f g initial]
+   (loop [i (f x)
```
Comment (tclamb) [inline src/qwircl/grid.cljs]: I think this `loop` can be rewritten as `iterate` with `take-while`.

File: src/qwircl/grid.cljs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          j (g y)
+          acc initial]
+     (if-let [cell (get-in grid [i j])]
+       (recur (f i) (g j) (conj acc cell))
+       acc)))
+  ([grid clicked direction]
+   (let [[f1 f2 g1 g2] (map #(get-in transformations (apply (partial conj [direction]) %))
```
Comment (tclamb) [inline src/qwircl/grid.cljs]: `(apply (partial conj [...]) %)` is equivalent to `(into [...] %)` (also `(apply vector ... %)`, but I think `into` is more idiomatic here)

**Review by tclamb — APPROVED** [review]
👍👍

File: src/qwircl/grid.cljs (hunk 1 above)
Comment (ivansalazar) [reply src/qwircl/grid.cljs]: Lemme see.

File: src/qwircl/grid.cljs (hunk 2 above)
Comment (ivansalazar) [reply src/qwircl/grid.cljs]: Good point.

**Comment by ivansalazar** [comment]
OK, one more commit before pushing if there are no more comments.

**Pull request merged**